concordium-sdk/src/main/java/com/concordium/sdk/responses/poolstatus/PoolStatus.java

```diff
@@ -0,0 +1,35 @@
+package com.concordium.sdk.responses.poolstatus;
+
+import com.concordium.sdk.serializing.JsonMapper;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import concordium.ConcordiumP2PRpc;
+import lombok.Data;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.val;
+
+@Data
+@Getter
+@ToString
```

Comment:
I don't see the need for `@Getter` and `@ToString` when `@Data` is being used.

Reply:
done